exercises/stack_02.js

```diff
@@ -0,0 +1,167 @@
+//Stack.push: one
+window.Collections = Object.create(null);
+
+(function(root) {
+    function Stack(arr) {
+        this.arr = arr;
+    };
+    Stack.prototype.push = function(elem, callback) {
+        this.arr.push(elem);
+        callback(this.arr);
```

Comment:
Your `push()` signature is wrong here. All you need is `function(elem) { ... }`... no callback necessary.